vcs_server/FileLoader.py

```diff
@@ -70,12 +75,12 @@ def variables(self, file_name):
                     outVars[lonName] = {}
                 lonData = var.getLongitude()[:]
                 outVars[lonName]['bounds'] =\
-                  [numpy.amin(lonData), numpy.amax(lonData)]
+                  [n2p(numpy.amin(lonData)), n2p(numpy.amax(lonData))]
```

Comment:
why not simply use `float(min(lonData))`?

Reply:
Yes that could possibly work and add less code.